Fmaraev/Hometask_1/palindrome.py

```diff
@@ -0,0 +1,14 @@
+x = int(input())
+n = 1
+k = 0
+while x // n != 0:
```

Comment:
this solution looks too complex. please try to find easier way in 3-5 lines with one while(or for) loop